po/ja.po

```diff
@@ -8315,24 +8402,54 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"```rust,editable\n"
+"fn duplicate<T: Clone>(a: T) -> (T, T) {\n"
+"    (a.clone(), a.clone())\n"
+"}\n"
+"\n"
+"// 以下の糖衣構文です：\n"
+"//   fn add_42_millions<T: Into<i32>>(x: T) -> i32 {\n"
+"fn add_42_millions(x: impl Into<i32>) -> i32 {\n"
+"    x.into() + 42_000_000\n"
+"}\n"
+"\n"
+"// struct NotClonable;\n"
+"\n"
+"fn main() {\n"
+"    let foo = String::from(\"foo\");\n"
+"    let pair = duplicate(foo);\n"
+"    println!(\"{pair:?}\");\n"
+"\n"
+"    let many = add_42_millions(42_i8);\n"
+"    println!(\"{many}\");\n"
+"    let many_more = add_42_millions(10_000_000);\n"
+"    println!(\"{many_more}\");\n"
+"}\n"
+"```"
 
 #: src/traits/trait-bounds.md:35
 msgid "Show a `where` clause, students will encounter it when reading code."
 msgstr ""
+"`where` 節を見せてみましょう。受講生はコードを読んでいるときに、この`where`節"
+"に遭遇します。"
 
 #: src/traits/trait-bounds.md:46
 msgid "It declutters the function signature if you have many parameters."
 msgstr ""
+"たくさんのパラメタがある場合に、`where`節は関数のシグネチャを整理整頓してくれ"
```

Comment:
たくさんのパラメタがある場合、関数シグネチャが煩雑になります。
which is the reason why people want to use `where` clause.

Reply:
Thanks! I would prefer describing positive outcome of using `where` rather than negative outcomes without using it as in the original sentence. Of course, we can also paraphrase that way, too!